testsuite/features/step_definitions/command_steps.rb

```diff
@@ -383,6 +383,19 @@
   end
 end
 
+When(/^I increase Java log level to "([^"]*)" on server$/) do |log_level|
+  $server.run(
+    "echo -e >>/srv/tomcat/webapps/rhn/WEB-INF/classes/log4j.properties << CONFIG
```

Comment:
no `-e` needed if you use `<<`

maybe `<<CONFIG` with no space (???)

Reply:
right, I "tested" this in a Ruby shell but without the space.... And you're right of course about the `-e`, no need to interpret backslashes if I don't have them in the string :D